gateway-sp-comms/src/single_sp.rs

```diff
@@ -1070,6 +1070,20 @@ impl AttachedSerialConsoleSend {
 
         rx.await.unwrap()
     }
+
+    pub async fn send_break(&self) -> Result<()> {
+        let (tx, rx) = oneshot::channel();
+        self.inner_tx
+            .send(InnerCommand::Rpc(RpcRequest {
+                kind: MgsRequest::SerialConsoleBreak,
+                our_trailing_data: None,
+                response_tx: tx,
+            }))
+            .await
+            .unwrap();
+        rx.await.unwrap();
```

Comment:
Couple comments on this function:

1. There's a free `rpc` function that helps with some of the boilerplate (creating the oneshot, etc.).
2. We should check that the SP sent back the expected `SerialConsoleBreakAck`.

Combining both of those, I think this should be roughly:

```rust
        rpc(&self.inner_tx, MgsRequest::SerialConsoleBreak, None)
            .await
            .result
            .and_then(|(_peer, response, _data)| {
                response.expect_serial_console_break_ack()
            })
```

which will require adding `expect_serial_console_break_ack()` to `SpResponseExt`.